### PR TITLE
Philanthropy Page

### LINKED ISSUE #18: Philanthropy Page
image
Take images from figma, use flexbox to make sure they are in the correct position
use button , title, and footer components

### PR DESCRIPTION
**Pull request by bgarcia11**
Screenshot_2024-10-16_05-07-29_0
-Added logos/images in public
-Added placeholders for button and footer components (so that it would compile)
-Added page component and imported it in app

### CODE REVIEW

File: public/philanthropy/IAWF.png
Comment (kavinphab) [inline public/philanthropy/IAWF.png:1]: convert to svg

File: public/philanthropy/childrenInternational.png
Comment (kavinphab) [inline public/philanthropy/childrenInternational.png:1]: convert to svg

**Comment by kavinphab** [comment]
image check failed (check for files in the public folder that don't belong there), page looks good though, make text a little smaller

**Review by kavinphab — CHANGES_REQUESTED** [review]

**Comment by bgarcia11** [comment]
image

- Changed files to pass image check
- Made text smaller

**Review by kavinphab — APPROVED** [review]

**Pull request merged**